saleor/graphql/account/mutations/base.py

```diff
@@ -93,19 +98,35 @@ class Arguments:
             required=True,
             description="Email of the user that will be used for password recovery.",
         )
+        redirect_url = graphene.String(
+            required=True,
+            description=(
+                "Url of storefront view which allow user reset password, "
```

Comment:
I would reword this a little bit. We could also provide an example here of expected format using the example.com domain.

"URL of a view where users should be redirected to reset the password. URL in RFC 1808 format e.g. <example>"

Ideally, we should have our own `URL` scalar that we could use across the entire API that would validate if provided URL is valid.